Codeinwp/neve
# Removes Nav Cart Title | New alignment control for HFB

### Summary

- changes alignment control for HFG components [UI]
- removes cart widget title from nav cart [FIX]

### Will affect visual aspect of the product

NO


### Test instructions


- Check that alignment controls still work
- Check if cart widget still has title.


Closes #514, #513.
